doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -166,31 +179,31 @@ Kubernetes objects and their desired state.
    * - Pod and workloads
      - Description
    * - `Pod <https://kubernetes.io/docs/concepts/workloads/pods/>`__
-     - Pod is a collection of containers that can run on a node. This resource is created by clients and scheduled onto
-       nodes.
+     - A pod is a collection of containers that can run on a node. This resource is created by clients
+       and scheduled onto nodes.
    * - `ReplicaSet <https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/>`__
-     - ReplicaSet ensures that a specified number of pod replicas are running at any given time.
+     - A ReplicaSet ensures that a specified number of pod replicas are running at any given time.
    * - `Deployment <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`__
-     - Deployment enables declarative updates for Pods and ReplicaSets.
+     - A deployment enables declarative updates for pods and ReplicaSets.
    * - `DaemonSet <https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/>`__
-     - A Daemon set ensures that the correct nodes run a copy of a Pod.
+     - A DaemonSet ensures that the correct nodes run a copy of a pod.
    * - `Job <https://kubernetes.io/docs/concepts/workloads/controllers/job/>`__
-     - A Job represent a task, it creates one or more Pods and will continue to retry until the expected number of
-       successful completions is reached.
+     - A job represents a task. It creates one or more pods and will continue to retry until the
+       expected number of successful completions is reached.
    * - `CronJob <https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/>`__
-     - A CronJob manages time-based Jobs, namely: once at a specified point in time and repeatedly at a specified point
-       in time.
+     - AA CronJob manages time-based jobs, namely, once at a specified time and repeatedly at specified times.
    * - `StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`__
-     - StatefulSet represents a set of pods with consistent identities. Identities are defined as: network, storage.
+     - A StatefulSet represents a set of pods with consistent identities. Identities are defined as network and storage.
 
 CPU Management
```

Comment:
should this be "CPU Management Policy"

Reply:
Fixed in [8eeb6e9](https://github.com/cntt-n/CNTT/pull/3217/commits/8eeb6e9c166cc3a4166238034dc17583f2e5fbac)